### PR TITLE
fix: move onHidden from hide to onDismiss

### PR DESCRIPTION
**Pull request by geekvijay**

### CODE REVIEW

**Comment by mxck** [comment]
@geekvijay hi! Did you check this on android? :)
https://facebook.github.io/react-native/docs/modal#ondismiss - platform `iOS`

**Comment by geekvijay** [comment]
I totally missed that, I will make the required changes and send you the pull request. :)

**Comment by geekvijay** [comment]
I have done the required changes :)

File: src/Menu.js [hunk 1]
```diff
@@ -106,14 +106,20 @@ class Menu extends React.Component {
...
+          if (Platform.OS !== 'ios' && this.props.onHidden) {
             this.props.onHidden();
           }
         },
       );
     });
   };
 
+  _onDismiss = () => {
+    if (Platform.OS === 'ios' && this.props.onHidden) {
```
Comment (mxck) [inline src/Menu.js]: You can remove platform check here.

Comment (geekvijay) [reply src/Menu.js]: done :)

**Comment by mxck** [comment]
@geekvijay thx!

**Comment by mxck** [comment]
@geekvijay I make release today.

**Comment by geekvijay** [comment]
Thanks for this library. We are using it in our project :)

**Comment by mxck** [comment]
@geekvijay released in 0.4.2

**Comment by mxck** [comment]
@geekvijay glad to hear that :)

**Pull request merged**